### PR TITLE
Add art assets / fixes #346

### LINKED ISSUE #346: Art assets are missing
Just a reminder for myself:

* picture of Spongie 
* official logo / Banner

are both still missing on the docs.
Specifically here: https://docs.spongepowered.org/en/about/assets.html

### PR DESCRIPTION
**Pull request by Tzky**
This adds:
*SpongePowered logo
*Spongie logo
*instructions about usage of the images

@Zidane and @Inscrutable 
I guess it's best if you both have a final look at this before merging. I guess that you're the ones who should decide which rights users have to use this images ;)

I hope the res of the images is fine (width around 500px). Or should we provide a link to highres (or vector) files too?

### CODE REVIEW

**Comment by Spongy** [comment]
Setting up PR reference, please wait...

**Comment by Tzky** [comment]
I'm not sure this is needed:
``Use the mascot as a link to the Sponge project only. The SpongePowered logo (which retains Spongie’s form) is a better representation of the Sponge project when used as an affiliate or reference.``

Thoughts?

**Comment by saladoc** [comment]
Better rephrase:
`Use the mascot as a link to the Sponge project. The SpongePowered logo (which retains Spongie's form) should be used instead as it is a better representation of the Sponge project.`

**Comment by Inscrutable** [comment]
This PR may get delayed as the Overlords consider consolidation of the style and appearance of the Docs, Forums, Ore, Webpage and Reddit. Even the banner isn't final yet. Now docs staff should have access to the assets on GDocs, so at least you can see what we *do* have...

**Comment by Tzky** [comment]
Note to self: Add note that english license counts.

File: source/about/assets.rst [hunk 1]
```diff
@@ -3,19 +3,51 @@ Art Assets
 ==========
 
 This page will provide the official logo and other art assets in the future. Feel free to use them to spread the word
-about Sponge.
+about Sponge. However note that these images **aren't** provided under the MIT License.
 
 .. note::
-  The provided graphics should be used for non-commercial purposes only. Please provide a link to this project when
-  using some of our art. e.g. `Sponge Homepage <http://spongepowered.org>`__
+  If you're reading a translated version, please note that the english license is the one which counts. Translated
```
Comment (RobertHerhold) [inline source/about/assets.rst]: Capital E in "English"

**Pull request merged**